ui/packages/label/src/Label.svelte

```diff
@@ -5,16 +5,28 @@
 	};
 
 	export let show_label: boolean;
+	export let color: string | undefined;
 </script>
 
 <div class="output-label">
-	<div
-		class:sr-only={!show_label}
-		class="output-class font-bold text-2xl py-6 px-4 flex-grow flex items-center justify-center dark:text-slate-200"
-		class:no-confidence={!("confidences" in value)}
-	>
-		{value.label}
-	</div>
+	{#if color !== null}
```

Comment:
What's the best practice to reduce this code dupe? The only difference is `style:background-color` but you can't place an {#if} around just the style

Reply:
You can create a conditional expression in the curly braces, falling back tot he 'default' which is probably transparent in this case (swap transparent for something else if that isn't the case).

```svelte
<div style:background-color={color || "transparent"}>
  ...
</div>

Reply:
Thank you for the tip @pngwn ! Works and makes it much cleaner!